examples/12-fluids/02-fluids_results.py

```diff
@@ -0,0 +1,251 @@
+# extracted for all face zones (excluding interior zones), and exported to a
+# Faces ``Field``. Face results defined on all face zones bring their
+# ``MeshSupport`` by default.
+
+print(rinfo.available_results[4])
+mdot = dpf.operators.result.mass_flow_rate(streams_container=streams).eval()
+print(mdot)
+print(mdot[0])
+print(mdot[0].meshed_region)
+pl = DpfPlotter()
+pl.add_field(mdot[0], mdot[0].meshed_region)
+cpos = [
+    (0.33174794676955766, -0.0004666003573986906, -0.003683572358464899),
+    (0.00308308291385862, 0.0018660501219530234, 0.002657631405566274),
+    (0.00837023599064669, -0.7165991168396113000, 0.697435047079045400),
+]
+pl.show_figure(cpos=cpos, show_axes=True)
+pl = DpfPlotter()
+pl.add_field(mdot[0], mdot[0].meshed_region)
+cpos = [
+    (-0.22143903910901550, -0.0048827077124325, 0.0037923111967477),
+    (0.00300595909357079, -2.622604370117e-06, 7.033083496094e-06),
+    (-0.02716069471785041, 0.6830445754068439, -0.7298714987377765),
+]
+pl.show_figure(cpos=cpos, show_axes=True)
+
+###############################################################################
+# As this result is defined for several zones, the region_scoping pin can be used
+# to extract a subset of them. We can get for example the mass flow rate for all
+# inlets and outlets of the model. Face results defined on individual zones need
+# the connection of the mesh pin to retrieve their right mesh_support. In particular,
+# the connected entity should be a ``MeshesContainer`` labelled on zone. This is the
+# output from the meshes_provider operator, as seen in :ref:`ref_fluids_mesh`
+in_sco = dpf.Scoping(ids=[3], location=dpf.locations.zone)
+in_meshes = dpf.operators.mesh.meshes_provider(
+    streams_container=streams, region_scoping=in_sco
+).eval()
+mdot_in = dpf.operators.result.mass_flow_rate(
+    streams_container=streams, region_scoping=in_sco, mesh=in_meshes
+).eval()
+print(mdot_in)
+
+out_sco = dpf.Scoping(ids=[4, 5, 6, 7], location=dpf.locations.zone)
+out_meshes = dpf.operators.mesh.meshes_provider(
+    streams_container=streams, region_scoping=out_sco
+).eval()
+mdot_out = dpf.operators.result.mass_flow_rate(
+    streams_container=streams, region_scoping=out_sco, mesh=out_meshes
+).eval()
+print(mdot_out)
+
+pl = DpfPlotter()
+pl.add_field(mdot_in[0], mdot_in[0].meshed_region)
+pl.add_field(mdot_out[0], mdot_out[0].meshed_region)
+pl.add_field(mdot_out[3], mdot_out[3].meshed_region)
+pl.add_field(mdot_out[6], mdot_out[6].meshed_region)
+pl.add_field(mdot_out[9], mdot_out[9].meshed_region)
+pl.show_figure(cpos=cpos, show_axes=True)
+
+###############################################################################
+# To filter a particular phase for a certain selection of zones, the qualifiers
+# pin can be used.
+
+mdot_out_prov = dpf.operators.result.mass_flow_rate(streams_container=streams, mesh=in_meshes)
+mdot_out_prov.connect(1000, {"zone": 4, "phase": 2})
```

Comment:
@rafacanton 
also, just a note, according to my linter the `.connect()` method does not recognize `dict` type as an acceptable argument type.